CodeEdit/Settings/Models/PreferencesIdentifier.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  PreferencesIdentifier.swift
+//  CodeEdit
+//
+//  Created by 朱浩宇 on 2022/3/23.
+//
+
+import Preferences
+
+extension Preferences.PaneIdentifier {
+    static let general = Self("general")
+    static let theme = Self("Theme")
+    static let execution = Self("execution")
```

Comment:
Why `execution` and not `terminal` ?

Reply:
execution is for forthcoming feature: running project. There will be another terminal tab